mmv1/third_party/terraform/services/storage/resource_storage_bucket_object.go

```diff
@@ -120,7 +122,7 @@ func ResourceStorageBucketObject() *schema.Resource {
 				Type: schema.TypeString,
 				// This field is not Computed because it needs to trigger a diff.
 				Optional: true,
-				ForceNew: true,
+				//ForceNew: true,
```

Comment:
no need to comment out code that you don't intend on using.

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/10038/commits/e06c803fde4de09549f2e1a678e1c06f005b502d